include/iarc7_fc_comms/CommonFcComms.hpp

```diff
@@ -124,16 +134,20 @@ namespace FcComms{
         // Subscriber for uav_angle values
         ros::Subscriber uav_angle_subscriber;
 
-        // Subscriber for uav_throttle valuess
-        ros::Subscriber uav_throttle_subscriber;
+        // Subscriber for the contact switch values
+        ros::Subscriber contact_switch_subscriber;
 
         // Service to arm copter
         ros::ServiceServer uav_arm_service;
 
         iarc7_msgs::OrientationThrottleStamped::ConstPtr last_direction_command_message_ptr_;
 
+        iarc7_msgs::LandingGearContactsStamped::ConstPtr last_contact_switch_message_ptr_;
+
         bool have_new_direction_command_message_ = false;
 
+        bool have_contact_switch_message_ = false;
```

Comment:
Don't need this, just test if `last_contact_switch_message_ptr_` is null.  We should also be keeping track of not only if we have a message, but if we have a recent one.